man/man4/zfs.4

```diff
@@ -2098,6 +2098,13 @@ Limit SLOG write size per commit executed with synchronous priority.
 Any writes above that will be executed with lower (asynchronous) priority
 to limit potential SLOG device abuse by single active ZIL writer.
 .
+.It Sy zfs_zil_saxattr Ns = Ns Sy 1 Ns | Ns 0 Pq int
+Setting this tunable to zero disables ZIL logging of new
+.Sy xattr Ns = Ns Sy sa
+records if feature org.openzfs:zilsaxattr is enabled on the pool. This would
+only be necessary to work around bugs in the ZIL logging or replay code for this
+record type. The tunable has no effect if the zpool feature org.openzfs:zilsaxattr is disabled.
```

Comment:
```
records if the
.Sy org.openzfs:zilsaxattr
feature is enabled on the pool.
This would only ... type.
The tunable has no effect if the feature is disabled.
```

Reply:
Done. Thanks!